### PR TITLE
Automation of knative-plugin | admin-perspective

### PR DESCRIPTION
**Pull request by SparshKesari**
**Description:**

- Automation of eventing-page-admin feature file.
- Automation of filters-serving-eventing-admin feature file.
- Automation of serverless-admin-empty-state feature file.
- Automation of serving-page-admins feature file.

**Jira Id:** 

- [ODC-6647](https://issues.redhat.com/browse/ODC-6647) - eventing-page-admin
- [ODC-6648](https://issues.redhat.com/browse/ODC-6648) - filters-serving-eventing-admin
- [ODC-6649](https://issues.redhat.com/browse/ODC-6649) - serverless-admin-empty-state
- [ODC-6650](https://issues.redhat.com/browse/ODC-6650) - serving-page-admins

**Pre-conditions/setup:**

**Checks for approving Epic scenarios Automation PR:**

- [x] Execute the @to-do tagged gherkin scripts manually
- [x] Convert the @to-do gherkin scripts to cypress automation scripts
- [x] Once scripts are automated, replace tag @to-do with @epic-number
- [x] Execute the scripts in Remote cluster

**Refactoring criteria for approving Automation PR:**

- [ ] update the test cases count in [Automation status confluence Report](https://docs.jboss.org/display/ODC/Automation+Status+Report)

**Execution Commands:**
Example: 
```
export NO_HEADLESS=true && export CHROME_VERSION=$(/usr/bin/google-chrome-stable --version)
    BRIDGE_KUBEADMIN_PASSWORD=YH3jN-PRFT2-Q429c-5KQDr
    BRIDGE_BASE_ADDRESS=https://console-openshift-console.apps.dev-svc-4.8-042801.devcluster.openshift.com
    export BRIDGE_KUBEADMIN_PASSWORD
    export BRIDGE_BASE_ADDRESS
    oc login -u kubeadmin -p $BRIDGE_KUBEADMIN_PASSWORD
    oc apply -f ./frontend/integration-tests/data/htpasswd-secret.yaml
    oc patch oauths cluster --patch "$(cat ./frontend/integration-tests/data/patch-htpasswd.yaml)" --type=merge
    ./test-cypress.sh -p knative

```

**Screen shots**:

- serving-page-admins feature file.
image

- serverless-admin-empty-state feature file.
image

- eventing-page-admin feature file.
image

- filters-serving-eventing-admin feature file.

**Browser conformance**:

- [x] Chrome
- [ ] Firefox
- [ ] Safari
- [ ] Edge

### CODE REVIEW

**Comment by jrichter1** [comment]
I'm facing some issues with the eventing page:
Screenshot from 2022-04-27 10-49-20
Screenshot from 2022-04-27 10-48-57

File: frontend/packages/knative-plugin/integration-tests/cypress.json [hunk 1]
```diff
@@ -19,7 +19,7 @@
   "screenshotsFolder": "../../../gui_test_screenshots/cypress/screenshots",
   "videosFolder": "../../../gui_test_screenshots/cypress/videos",
   "env": {
-    "TAGS": "@knative and (@pre-condition or @smoke or @regression) and not (@manual or @to-do or @un-verified or @broken-test)",
+    "TAGS": "(@pre-condition or @smoke or @regression) and not (@manual or @to-do or @un-verified or @broken-test)",
```
Comment (sanketpathak) [inline frontend/packages/knative-plugin/integration-tests/cypress.json:22]: Include the tags knative-admin tag instead of removing any tag

**Comment by sanketpathak** [comment]
Tests are failing on local
Screenshot from 2022-04-27 20-26-44

**Comment by jrichter1** [comment]
Tests are now green, everything seems to be in order.
/lgtm

**Comment by jrichter1** [comment]
This PR only affects our e2e tests.
/label px-approved
/label docs-approved
/label qe-approved

**Pull request merged**